prow/plugins/approve/approve.go

```diff
@@ -99,7 +99,7 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
 		}
 		opts := optionsForRepo(config, parts[0], parts[1])
-		approveConfig[repo] = fmt.Sprintf("Pull requests %srequire an associated issue.<br>Pull request authors %simplicitly approve their own PRs.", doNot(opts.IssueRequired), doNot(opts.ImplicitSelfApprove))
+		approveConfig[repo] = fmt.Sprintf("Pull requests %srequire an associated issue.<br>Pull request authors %simplicity approve their own PRs.", doNot(opts.IssueRequired), doNot(opts.ImplicitSelfApprove))
```

Comment:
This should be `implicitly` not `simplicity`.

Reply:
Oh right: this should probably be `%s implicitly` or as-is..